inc/template.php

```diff
@@ -52,10 +52,16 @@ function get_authors( WP_Post $post ) : array {
 	}
 
 	/** @var WP_User[] */
-	$users = get_users( [
-		'include' => $author_ids,
-		'orderby' => 'include',
-	] );
+	$users = wp_cache_get( 'authors', 'authorship' );
```

Comment:
@addaeabeng this means all posts will share the same cached users, meaning it won't be possible to have more than one post on a site, the cache needs to be per-post